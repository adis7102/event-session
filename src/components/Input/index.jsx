import React from "react";

const Input = (props) => {
  const { name, title, onChange, value } = props;

  return (
    <div className="input">
      <p className="title">{title}</p>
      <input
        type="text"
        name={name}
        onChange={(e) => onChange(e?.target?.value)}
        value={value}
        data-testid="search-input"
      />
    </div>
  );
};

export default Input;
