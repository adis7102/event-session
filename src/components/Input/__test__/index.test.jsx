import React from "react";
import Input from '../index';

import { render } from "@testing-library/react";

it('should render Input correctly', () => {
    const { container } = render(<Input />);
  
    expect(container).toMatchSnapshot();
});