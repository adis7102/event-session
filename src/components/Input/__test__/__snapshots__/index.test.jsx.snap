// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Input correctly 1`] = `
<div>
  <div
    class="input"
  >
    <p
      class="title"
    />
    <input
      data-testid="search-input"
      type="text"
      value=""
    />
  </div>
</div>
`;
